earthpy/spatial.py

```diff
@@ -161,16 +161,11 @@ def crop_image(raster, geoms, all_touched = True):
         Specifically the extent (shape elements) and transform properties are updated.
     """
 
-    # test that you have a list of a geodataframe
```

Comment:
@mlevis1 i believe this was already done last week and merged. Did you update your fork before working on this? @jlpalomino  just giving you a heads  up!